recipes/mizuroute/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+  ln -s "${CC}" "${BUILD_PREFIX}/bin/gcc"
+  ln -s "${FC}" "${BUILD_PREFIX}/bin/gfortran"
```

Comment:
Please do not do this, but instead modify the makefile to use the CC and FC variables

Reply:
Fixed this